2019-12-04 15:09:49.447 [restartedMain] WARN  org.springframework.cloud.config.client.ConfigServicePropertySourceLocator - Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/api-swagger/default": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
2019-12-04 15:11:39.154 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2019-12-04 15:11:39.182 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2019-12-04 15:12:05.419 [restartedMain] WARN  org.springframework.boot.devtools.autoconfigure.OptionalLiveReloadServer - Unable to start LiveReload server
2019-12-04 15:13:02.878 [RMI TCP Connection(3)-192.168.30.1] WARN  org.springframework.cloud.config.client.ConfigServicePropertySourceLocator - Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/api-swagger/default": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
2019-12-04 15:22:27.372 [http-nio-10000-exec-1] WARN  org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver - Resolved exception caused by Handler execution: com.netflix.hystrix.exception.HystrixRuntimeException: CrunchiesService#list() timed-out and fallback failed.
2019-12-04 15:23:43.136 [http-nio-10000-exec-6] WARN  org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver - Resolved exception caused by Handler execution: com.netflix.hystrix.exception.HystrixRuntimeException: CrunchiesService#list() timed-out and fallback failed.
